### PR TITLE
 [ACM-2.11] ACM#11544: Remove the  hypershift CLI mentions

### PR DESCRIPTION
**Pull request by xenolinux**
https://issues.redhat.com/browse/ACM-11544

### CODE REVIEW

**Comment by zanetworker** [comment]
thanks Serveesha, now we need to make sure these commands exist in `hcp` cli. I.e., it might not be a simple replace.

**Comment by xenolinux** [comment]
> thanks Serveesha, now we need to make sure these commands exist in `hcp` cli. I.e., it might not be a simple replace.

@zanetworker Sure. I will use this PR and issue to track any changes related to that. 

Edit: will revisit the need for new jira per findings though

File: clusters/hosted_control_planes/deploy_aws_private_clusters.adoc [hunk 1]
```diff
@@ -107,20 +107,12 @@ You can access a private cluster by using a bastion instance.
 . Start a bastion instance by entering the following command:
 +
 ----
-hypershift create bastion aws --aws-creds=<aws-creds> --infra-id=<infra-id> --region=<region> --ssh-key-file=<ssh-key>
+hcp create bastion aws --aws-creds=<aws-creds> --infra-id=<infra-id> --region=<region> --ssh-key-file=<ssh-key>
```
Comment (bryan-cox) [inline clusters/hosted_control_planes/deploy_aws_private_clusters.adoc]: HCP CLI doesn't currently support `hcp create bastion`

File: clusters/hosted_control_planes/manage_aws_infra_iam.adoc [hunk 2]
```diff
@@ -473,7 +473,7 @@ To create the cloud infrastructure portion separately, you need to create the AW
 To create the AWS infrastructure, enter the following command:
 //lahinson - july 2023 - update hypershift cli command here
 ----
-hypershift create infra aws --name CLUSTER_NAME \ <1>
+hcp create infra aws --name CLUSTER_NAME \ <1>
```
Comment (bryan-cox) [inline clusters/hosted_control_planes/manage_aws_infra_iam.adoc]: HCP CLI doesn't currently support `hcp create infra aws`

File: clusters/hosted_control_planes/manage_aws_infra_iam.adoc [hunk 3]
```diff
@@ -516,7 +508,7 @@ All of those resources contain the `kubernetes.io/cluster/INFRA_ID=owned` tag, w
 To create the AWS IAM resources, enter the following command:
 //lahinson - july 2023 - update hypershift cli command here
 ----
-hypershift create iam aws --infra-id INFRA_ID \ <1>
+hcp create iam aws --infra-id INFRA_ID \ <1>
```
Comment (bryan-cox) [inline clusters/hosted_control_planes/manage_aws_infra_iam.adoc:518]: HCP CLI doesn't currently support `hcp create iam aws`

**Comment by zanetworker** [comment]
@xenolinux for the things that are not supported in the hcp cli today, we should then remove their pieces from the docs and add the requirements / what needs to be there till we have the commands up.

**Comment by xenolinux** [comment]
Closed this PR since this change requires an additional information along with removal of the unsupported `hcp` flags.

**Pull request closed without merging**